django_celery_token_bucket/bucket.py

```diff
@@ -35,6 +36,13 @@ def get_queue(self):
             durable=True,
         )
 
+    def _get_or_create_interval(self):
+        from django_celery_beat.models import IntervalSchedule
+        from django_celery_beat.models import SECONDS
+
+        intervalschedule, created = IntervalSchedule.objects.get_or_create(period=SECONDS, every=self.interval_in_seconds)
```

Comment:
Can you please use `_created` to indicate that the variable is not used.
See https://pylint.pycqa.org/en/latest/user_guide/configuration/all-options.html#dummy-variables-rgx

Reply:
done